minimesos/src/main/java/com/containersol/minimesos/docker/DockerContainersUtil.java

```diff
@@ -102,9 +103,24 @@ public void remove() {
      * Removes all containers in the util object
      */
     public DockerContainersUtil kill() {
+        return kill(false);
+    }
+
+    /**
+     * Removes all containers in the util object
+     *
+     * @param ignoreFailure - use <code>true</code> if you expect containers might be stopped by this time
+     */
+    public DockerContainersUtil kill(boolean ignoreFailure) {
         if (containers != null) {
             for (Container container : containers) {
-                dockerClient.killContainerCmd(container.getId()).exec();
+                try {
+                    dockerClient.killContainerCmd(container.getId()).exec();
+                } catch (DockerException failure) {
+                    if(!ignoreFailure) {
```

Comment:
This ignores all failures (not only that a container did not exist). Is this intended?